client/src/components/client-page/index.jsx

```diff
@@ -0,0 +1,232 @@
+import Box from "@mui/material/Box";
+import Paper from "@mui/material/Paper";
+import Chip from "@mui/material/Chip";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import ArrowBackIcon from "@mui/icons-material/ArrowBack";
+import Typography from "@mui/material/Typography";
+import { Avatar } from "@mui/material";
+import Grid from "@mui/material/Grid";
+import Divider from "@mui/material/Divider";
+
+export default function ClientPage() {
+  const userInfo = {
+    firstName: "First Last",
+    email: "email@email.com",
+    phone: "+1 111 111 1111",
+    status: "Active"
+  };
+
+  // Just for now
+  const handleDelete = () => {
+    console.info("You clicked the edit icon.");
+  };
+
+  return (
+    <div style={{ marginLeft: "40px", marginRight: "40px" }}>
+      <div
+        style={{ display: "flex", alignItems: "center", marginBottom: "10px" }}
+      >
+        <ArrowBackIcon sx={{ color: "gray", marginRight: 2, marginLeft: 2 }} />
+        <Box className="title">
+          <Typography marginBottom={1} variant="h4" align="left">
+            John Smith
+          </Typography>
+          <Typography variant="subtitle2" align="left">
+            Client
+          </Typography>
+        </Box>
+        <Card id="people-card" sx={{ marginLeft: "auto" }}>
+          <CardContent>
+            <Grid container spacing={1}>
+              <Grid
+                item
+                xs={6}
+                sx={{ textAlign: "center", fontFamily: "Arial" }}
+              >
+                Owner
+              </Grid>
+              <Grid
+                item
+                xs={6}
+                sx={{ textAlign: "center", fontFamily: "Arial" }}
+              >
+                Creator
+              </Grid>
+
+              <Grid item xs={6}>
+                <Chip
+                  variant="outlined"
+                  avatar={
+                    <Avatar
+                      sx={{
+                        background: "#E53568",
+                        color: "#FFFFFF",
+                        textAlign: "center",
+                        justifyContent: "center",
+                        alignItems: "center",
+                        display: "flex",
+                      }}
+                    >
+                      OP
+                    </Avatar>
+                  }
+                  label="Owen Perth"
+                  onDelete={handleDelete}
+                  deleteIcon={<EditIcon />}
+                />
+              </Grid>
+              <Grid item xs={6}>
+                <Chip
+                  variant="outlined"
+                  avatar={
+                    <Avatar sx={{ background: "blue", color: "white" }}>
+                      EG
+                    </Avatar>
+                  }
+                  label="Emily Gale"
+                />
+              </Grid>
+            </Grid>
+          </CardContent>
+        </Card>
```

Comment:
Change the layout here to closer match the one in the figma. Also change the colouring of the 
initials to white so that they are clearer. This is what it looks like on my end at the moment: 
<img width="407" alt="Screenshot 2024-01-09 at 7 23 19 PM" src="https://github.com/uoftblueprint/tces/assets/58835213/b910d7ca-3d1d-44ee-b911-3e509b0ef095">